static/js/ui_init.js

```diff
@@ -459,6 +460,7 @@ export function initialize_everything() {
 
     const user_status_params = pop_fields("user_status");
 
+    about_zulip.initialize();
```

Comment:
This should definitely not be the first module initialized; modules that are initialized early are done like that for a reason (E.g. because we need them initialized before other things).  I moved this to be initialized last.

Reply:
I determined this needed to be before `hashchange.js`, so I moved it to that position and added an explanatory comment.